packages/swc-plugin-transform-cx-jsx/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@swc/plugin-transform-cx-jsx",
```

Comment:
I'm not sure if we will be able to publish a package under the `@swc` namespace.

Perhaps `swc-plugin-transform-cx-jsx`?